From adbdd4efafb5aae64def7fb8ae78a565d239039f Mon Sep 17 00:00:00 2001
From: Bandan Das <bsd@redhat.com>
Date: Tue, 15 Oct 2013 17:11:01 +0200
Subject: [PATCH 24/25] pc_piix: disable mixer for 6.4.0 machine types and below

RH-Author: Bandan Das <bsd@redhat.com>
Message-id: <1381857067-9399-5-git-send-email-bsd@redhat.com>
Patchwork-id: 54952
O-Subject: [PATCH RHEL-7 qemu-kvm v3 4/5] pc_piix: disable mixer for 6.4.0 machine types and below
Bugzilla: 954195
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

Signed-off-by: Bandan Das <bsd@redhat.com>
---
 hw/i386/pc_piix.c | 12 ++++++++++++
 1 file changed, 12 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/i386/pc_piix.c |   12 ++++++++++++
 1 files changed, 12 insertions(+), 0 deletions(-)

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index af1f602..3f5b913 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -925,6 +925,18 @@ static QEMUMachine pc_machine_rhel650 = {
         .driver   = "virtio-scsi-pci",\
         .property = "vectors",\
         .value    = stringify(2),\
+    },{\
+        .driver   = "hda-micro",\
+        .property = "mixer",\
+        .value    = "off",\
+    },{\
+        .driver   = "hda-duplex",\
+        .property = "mixer",\
+        .value    = "off",\
+    },{\
+        .driver   = "hda-output",\
+        .property = "mixer",\
+        .value    = "off",\
     }
 
 static void pc_init_rhel640(QEMUMachineInitArgs *args)
-- 
1.7.1

